src/validations/html/rules/index.js

```diff
@@ -0,0 +1,50 @@
+import Validator from '../../Validator';
+import runRules from '../runRules';
+import Code from './Code';
+import MismatchedTag from './MismatchedTag';
+
+const errorMap = {
+  [Code.MISPLACED_CLOSE_TAG]: ({openTag, closeTag, match}) => ({
+    reason: 'misplaced-close-tag',
+    payload: {
+      open: openTag.name,
+      close: closeTag.name,
+      misplacement: match.row + 1,
+    },
+  }),
+  [Code.UNCLOSED_TAG]: ({openTag, closeTag}) => ({
+    reason: 'unclosed-tag-by-mismatched-close-tag',
```

Comment:
Is there a reason not to just report this as a normal `unclosed-tag`? Is it really useful to include information about where we found a mismatched closing tag? Would it make more sense to still locate the error at the location of the opening tag? (These are all sincere/non-rhetorical questions!)